brightnessctl.c

```diff
@@ -535,18 +532,30 @@ int read_class(struct device **devs, char *class) {
 			continue;
 		}
 		devs[cnt++] = dev;
+		if (!read_all)
+			goto cleanup;
```

Comment:
IMHO: the goto is overkill here, a simple `break` should be enough

Reply:
Thanks! Nice catch